<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>140</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Должник</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#add_btn{
	border: 1px solid gray;
	border-radius: 7px;
	background-color: whitesmoke;
}

#add_btn:hover{
	background-color: #ededed;
}</string>
  </property>
  <widget class="QLineEdit" name="name_value">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 7px; border: 1px solid gainsboro; padding: 5px</string>
   </property>
   <property name="placeholderText">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QComboBox" name="priority_value">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Приоритет</string>
   </property>
   <item>
    <property name="text">
     <string>Обычный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Средний</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="add_btn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
